### PR TITLE
feat(docs): add new documentation site using nextra

### PR DESCRIPTION
**Pull request by maeertin**
Related to #52

### CODE REVIEW

**Comment by adamsoderstrom** [comment]
_Great initiative, @maeertin!_

File: apps/docs/LICENSE
Comment (adamsoderstrom) [inline apps/docs/LICENSE:1]: remove

File: apps/docs/src/pages/index.mdx
Comment (adamsoderstrom) [inline apps/docs/src/pages/index.mdx:1]: Rephrase

File: apps/docs/src/pages/about.mdx
Comment (adamsoderstrom) [inline apps/docs/src/pages/about.mdx:1]: Could probably be removed for now

File: docs/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "type": "git",
+    "url": "https://github.com/noaignite/accelerator.git",
+    "directory": "apps/docs"
+  },
+  "author": "NoA Ignite",
+  "type": "module",
+  "scripts": {
+    "build": "pnpm generate:markdown && next build",
+    "dev": "next dev",
+    "generate:markdown": "node --loader ts-node/esm --no-warnings=ExperimentalWarning ./scripts/generateMarkdown.ts",
```
Comment (adamsoderstrom) [inline docs/package.json:18]: **praise:** so nice!

**Review by adamsoderstrom — CHANGES_REQUESTED** [review]

**Review by adamsoderstrom — CHANGES_REQUESTED** [review]
Nice work, @maeertin !
Especially that we got automatic markdown generation from the JSDocs!

Though, one thing i think we should do move the `docs` directory into the `apps` directory.

A thought i have (non-blocking): Do we need to git track the generated Markdown files?

**Review by adamsoderstrom — APPROVED** [review]
We'll leave the documentation git-tracked for now.
If we stumble upon issues with lost documentation down the line, we'll take it then.

Good job! 🎉

**Comment by maeertin** [comment]
> Nice work, @maeertin ! Especially that we got automatic markdown generation from the JSDocs!
> 
> Though, one thing i think we should do move the `docs` directory into the `apps` directory.
> 
> A thought i have (non-blocking): Do we need to git track the generated Markdown files?

Tracking the generated mdx files for now as one might want to add manually written mdx files alongside the generated ones. We can change this behaviour later on as you say.

**Pull request merged**